plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -134,6 +136,21 @@ public void testReadFromView()
         onRemoteDatabase().execute("DROP VIEW IF EXISTS test_view");
     }
 
+    @Override
+    public void testReadMetadataWithRelationsConcurrentModifications()
+    {
+        assertThatThrownBy(
+                () -> {
+                    // The failure usually happens on the first iteration.
+                    for (int i = 0; i < 20; i++) {
+                        super.testReadMetadataWithRelationsConcurrentModifications();
+                    }
+                })
+                // TODO https://github.com/trinodb/trino/issues/10846: shouldn't fail
+                .hasMessageMatching("(?s).*(No task completed before timeout|was deadlocked on lock resources with another process and has been chosen as the deadlock victim).*");
+        throw new SkipException("to be fixed");
```

Comment:
Did you mean to have some assertions and also a SkipException? Seems like you want one or the other

Reply:
was intentional